packages/api/lib/granules.js

```diff
@@ -234,10 +234,10 @@ function getTotalHits(bodyHits) {
 }
 
 /**
- * Returns an array of granules from ElasticSearch query
+ * Returns an array of granules from an ElasticSearch query
  *
  * @param {Object} payload
- * @param {string} [payload.index] - ES index to query
+ * @param {string} [payload.index] - ES (metrics) index to query
```

Comment:
Same comment as above. Maybe clarify what 'metrics' means in this context.

Reply:
this was done in a few places.